src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,151 @@
+        System.out.println(Line);
+        System.out.println("     Hello! I'm Duke");
+        System.out.println("     What can I do for you?");
+        System.out.println(Line);
+    }
+
+    private static void writeToFile(String filePath) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        for(int i=0; i<taskSum; i++){
+            Task now = tasks.get(i);
+            fw.write(String.format("%d ",i+1));
+            fw.write(now.toString());
+            fw.write("\n");
+        }
+        fw.close();
+    }
+
+    public static void main(String[] args) {
+        greeting();
+        String command;
+        Scanner in = new Scanner(System.in);
+        String f = "data/lines.txt";
+        do{
+            command = in.nextLine();
+            try {
+                writeToFile(f);
+            } catch (IOException e) {
+                System.out.println("Something went wrong: " + e.getMessage());
+            }
+            if(command.contains("done")){
+                String[] number = command.split(" ");
+                int taskIndex = Integer.parseInt(number[1]) - 1;
+                tasks.get(taskIndex).complete();
+                System.out.println(Line + "\n" + "     Nice! I've marked this task as done:" + "\n     " +  tasks.get(taskIndex).toString() + "\n" + Line);
+            }
+            else if(command.contains("delete")){
+                String[] number = command.split(" ");
+                int taskIndex = Integer.parseInt(number[1]) - 1;
+                Task temp = tasks.get(taskIndex);
+                System.out.println(Line + "\n" + "     Noted. I've removed this task: " + "\n     " +  tasks.get(taskIndex).toString() + "\n" + "     Now you have " + taskSum + " tasks in the list");
+                System.out.println(Line);
+                tasks.remove(taskIndex);
+                taskSum--;
+            }
+            else if(command.contains("todo")){
+                int first = command.indexOf(" ");
+                int check = command.indexOf("d");
+                if(check == command.length()-2){
+                    System.out.println(Line);
+                    System.out.println("     ☹ OOPS!!! The description of a todo cannot be empty.");
+                    System.out.println(Line);
+                    continue;
+                }
+                String item = command.substring(first,command.length());
+                Task temp = new Todo(item);
+                tasks.add(temp);
+                taskSum++;
+                System.out.println(Line + "\n" + "     Got it. I've added this task: " + "\n" + "       [T][ ] " + item + "\n" + "     Now you have " + taskSum + " tasks in the list");
+                System.out.println(Line);
+                
+            }
+            else if(command.contains("deadline")){
+                int first = command.indexOf(" ");
+                int check = command.indexOf("deadline");
```

Comment:
Consider using a better variable name that is more descriptive. e.g. indexOfDeadline